packages/get-platform/src/getPlatform.ts

```diff
@@ -313,22 +348,22 @@ export async function getPlatform(): Promise<Platform> {
 
   // when the platform is linux
   if (platform === 'linux' && distro && libssl) {
-    return (distro + '-openssl-' + libssl) as Platform
+    return `${distro}-openssl-${libssl}` as Platform
   }
 
   // if just OpenSSL is known, fallback to debian with a specific libssl version
   if (libssl) {
-    return ('debian-openssl-' + libssl) as Platform
+    return `${defaultDistro}-openssl-${libssl}`
   }
 
   // if just the distro is known, fallback to latest OpenSSL 1.1
   if (distro) {
-    return (distro + '-openssl-1.1.x') as Platform
+    return `${distro}-openssl-${defaultLibssl}` as Platform
   }
 
   // use the debian build with OpenSSL 1.1 as a last resort
   // TODO: perhaps we should default to 'debian-openssl-3.0.x'
```

Comment:
When do we add `as Platform` and when do we not?

Reply:
We have to cast it to `Platform` when the inferred type of the expression is `string`. We could make it more type safe by making `distro` a union type of known distros instead of a string, and make the TS compiler check that the built platform name is actually included in the `Platform` type and force us to handle unknown distros specially.

Reply:
I think it's a bit out of scope of this PR though

Reply:
Yeah, I was just wondering based on some of them changing from old to new.